### PR TITLE
search genre

### PR DESCRIPTION
**Pull request by DongHien0896**
screen search
create ui: https://edu-redmine.framgia.vn/issues/1312
get data: https://edu-redmine.framgia.vn/issues/1314

### CODE REVIEW

**Comment by DongHien0896** [comment]
ready

File: app/src/main/java/com/framgia/hien/moviedb/screen/main/MainViewModel.java [hunk 1]
```diff
@@ -8,22 +8,33 @@
...
 import com.framgia.hien.moviedb.screen.favorite.FavoriteFragment;
 import com.framgia.hien.moviedb.screen.home.HomeFragment;
+import com.framgia.hien.moviedb.screen.home.HomeFragmentViewModel;
+import com.framgia.hien.moviedb.screen.search.SearchFragment;
 import com.framgia.hien.moviedb.util.network.NetworkReceiver;
 
-public class MainViewModel extends BaseViewModel implements BottomNavigationView.OnNavigationItemSelectedListener {
+import java.util.zip.GZIPOutputStream;
+
+public class MainViewModel extends BaseViewModel implements BottomNavigationView.OnNavigationItemSelectedListener, SearchView.OnQueryTextListener, MenuItem.OnActionExpandListener, HomeFragmentViewModel.OnClickSearchMoviesByGenre {
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/hien/moviedb/screen/main/MainViewModel.java]: vượt 80 kí tự

File: app/src/main/java/com/framgia/hien/moviedb/screen/play/PlayViewModel.java [hunk 2]
```diff
@@ -47,15 +49,20 @@ private void setComponent() {
...
+        mChangeListener = new PlayerStateChangeListener();
     }
 
     private void playvideo(final String key) {
         youtubeField.set(new YouTubePlayer.OnInitializedListener() {
             @Override
-            public void onInitializationSuccess(YouTubePlayer.Provider provider, YouTubePlayer youTubePlayer, boolean b) {
-                youTubePlayer.loadVideo(key);
-                mYouTubePlayer = youTubePlayer;
+            public void onInitializationSuccess(YouTubePlayer.Provider provider, YouTubePlayer youTubePlayer, boolean wasRestored) {
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/hien/moviedb/screen/play/PlayViewModel.java]: vượt 80 kí tự

File: app/src/main/java/com/framgia/hien/moviedb/screen/search/SearchFragment.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    public static SearchFragment getInstance(){
+        if (sFragment == null){
+            sFragment = new SearchFragment();
+        }
+        return sFragment;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/hien/moviedb/screen/search/SearchFragment.java]: vượt 80 kí tự

**Comment by DongHien0896** [comment]
ready

**Comment by DongHien0896** [comment]
ready

**Pull request merged**